src/pal/tests/CMakeLists.txt

```diff
@@ -26,7 +26,12 @@ else()
   message(FATAL_ERROR "Only ARM, ARM64 and AMD64 is supported")
 endif()
 
+# C++ emits errors and warnings for c-string literal fed into char* parameter
+# this is just to take care of the warnings
+add_compile_options(-Wno-writable-strings)
```

Comment:
Do we have this issue at many places? I would prefer fixing the code if it happens at only a few spots.